mapie/conformity_scores/conformity_scores.py

```diff
@@ -63,38 +69,48 @@ def get_signed_conformity_scores(
 
         Parameters
         ----------
-        y: NDArray
-            Observed values.
+        X: ArrayLike
+            Observed feature values.
 
-        y_pred: NDArray
-            Predicted values.
+        y: ArrayLike
+            Observed target values.
+
+        y_pred: ArrayLike
+            Predicted target values.
 
         Returns
         -------
         NDArray
-            Unsigned conformity scores.
+            Signed conformity scores.
         """
 
     @abstractmethod
     def get_estimation_distribution(
         self,
+        X: ArrayLike,
         y_pred: ArrayLike,
-        conformity_scores: ArrayLike,
+        values: ArrayLike
```

Comment:
I suggest keeping the previous term: `conformity_scores` to avoid confusion in understanding the method.

Reply:
Done